cmake/genmsg-extras.cmake.em

```diff
@@ -6,7 +6,7 @@ endif()
 set(_GENMSG_EXTRAS_INCLUDED_ TRUE)
 
 # set destination for langs
-set(GENMSG_LANGS_DESTINATION "etc/ros/genmsg")
```

Comment:
Have you tried the patch locally to ensure that it works?

I tried it with a simple package calling `find_package(genmsg)` and the variable is not defined in that line yet. Afaik this line must be moved below the `find_package(catkin REQUIRED)` call.

Reply:
Updated and tested, thanks for noticing!